services/s3/src/it/java/software/amazon/awssdk/services/s3/S3IntegrationTestBase.java

```diff
@@ -92,5 +97,4 @@ protected static String getBucketName(Class<?> testClass) {
     protected String getBucketName() {
         return BUCKET_NAMING_STRATEGY.getBucketName(getClass());
```

Comment:
Should this be removed then in favor of the new utility?

Reply:
Yes - I did have that on my TODO list - done now.